src/driver.jl

```diff
@@ -0,0 +1,53 @@
+struct Graph500Results{T<:Integer}
+    g::Graph{T}
```

Comment:
This is going to make this structure huge. Consider a dict `g => Graph500Results` where the graph itself is the key. That way you can keep the results small.